src/geos.c

```diff
@@ -19,14 +19,14 @@ int init_geos(PyObject* m) {
   PyModule_AddObject(m, "ShapelyError", base_class);
   geos_exception[0] = PyErr_NewException("shapely.errors.GEOSException", base_class, NULL);
   PyModule_AddObject(m, "GEOSException", geos_exception[0]);
-  return 0;
 
   void* context_handle = GEOS_init_r();
   // TODO: the error handling is not yet set up for the global context (it is right now
   // only used where error handling is not used)
   // GEOSContext_setErrorMessageHandler_r(context_handle, geos_error_handler, last_error);
   geos_context[0] = context_handle;
 
+  return 0;
```

Comment:
The reason this didn't actually cause issues (i.e. the global `geos_context` actually being null) is because this is right now only used for `GEOSGeom_Destroy`, and that function only needs the context in case an error happens when deallocating, which I suppose doesn't happen in standard usage)